src/app.js

```diff
@@ -160,7 +160,7 @@ app.routeAsync("/charon/getSourceInfo")
 app.routeAsync("/charon/*")
   .all((req) => {
     utils.warn(`(${req.method}) ${req.url} has not been handled / has no handler`);
-    throw new NotFound();
+    throw new httpErrors.httpErrors.NotFound();
```

Comment:
Doubling of property lookup here and a few other places.

Also, these exception classes are used so extensively in the codebase and are a fundamental part of the unit of computation (responses) that I would prefer shorter symbols than `httpErrors.NotFound()`. That could mean boilerplate like:

```js
import httpErrors from 'http-errors';
const {Forbidden, NotFound, Unauthorized} = httpErrors;
```

or wrapping that up into our own little module to ease the boilerplate.

Reply:
Whoops, good catch on my bulk-editing error! I'll look into adding a wrapper module since this import is used in quite a few places.

Reply:
Done: 93d5b248c4bf0f421aec4c8b254530d7e18da79a